.gitlab-ci.yml

```diff
@@ -119,123 +210,55 @@ generator:gource:
     - docker run -v "$PWD":/visualization $CI_REGISTRY/r3/docker/gource
   artifacts:
     paths: ['output.gif']
-  <<: *global_settings_comprehensive
+  <<: *global_on_build_doc
 
-# Documentation tests
-# --------------------------------------
+#
+# Run documentation tests
+#
 
-doc-tests:v1.6:
-  stage: documentation
+doc-tests:julia1.6:
+  stage: test
   image: $CI_REGISTRY/r3/docker/julia-custom
   script:
     - julia --project=@. -e 'import Pkg; Pkg.instantiate();'
     - julia --project=@. --color=yes test/doctests.jl
-  <<: *global_settings_pull
+  <<: *global_on_full_tests
 
-# Deploy the documentation
-# --------------------------------------
+#
+# Build and deploy the documentation
+#
 
 pages:
   stage: documentation
   dependencies:
-    - generator:gource
+    - docdata:gource
+      # Note: This dependency is also implied by the stage ordering, but let's
+      # be sure. As of Nov 2021, the assets are not used directly, but referred
```

Comment:
and this will stay like that as we do not have the possibility to store the assets on github (unless in an external repo)

Reply:
yeah but I felt that I should write this down to my future self who forgot that already... :D